### ISSUE TITLE
[Rule suggestion] Indentation for closure end

### ISSUE THREAD

**Issue by NachoSoto** [realm/SwiftLint#326]
Xcode still doesn't know how to indent this one correctly, and I'd love it if `SwiftLint` could detect it AND correct it, because it really kills my OCD.

I think this only happens when the closure starts in a "overly" indented line. A very common example of this is when using a chained operator, like in RAC:

``` swift
SignalProducer(values: [1, 2, 3])
    .map { $0 + 1 }
    .startWithNext { number in
        print(number)
}
```

That's how Xcode indents the closing `}`, which is really annoying. Instead, it should be aligned:

``` swift
SignalProducer(values: [1, 2, 3])
    .map { $0 + 1 }
    .startWithNext { number in
        print(number)
    }
```

I'm happy to implement this myself (I'd love to finally dive in `SwiftLint`'s source), just wanted to make sure other people agree with this too :)

**Comment by keith** [comment]
Totally agree with this. This is part of our styleguide as well.

**Comment by jpsim** [comment]
Looks good to me.
